backend/src/appointment/utils.py

```diff
@@ -176,6 +185,32 @@ def normalize_secrets():
     """Normalizes AWS secrets for Appointment"""
 
     log.info('Normalizing secrets...')
+
+    database_secrets = os.getenv('DATABASE_SECRETS')
```

Comment:
If `get_database_url` runs after `normalize_secrets` I think we can re-wire them up to the new env vars. Like 

```suggestion
    database_secrets = os.getenv('DATABASE_SECRETS')
    if database_secrets:
        secrets = json.loads(database_secrets)
        os.environ['DATABASE_HOST'] = secrets['host']
        ... etc ...
```

Reply:
It should. normalize_secrets runs out of __main__ on both API startup and CLI command execution. get_database_engine_and_session is when the various `DATABASE_*` settings get used. I'll see if I can wire that up a bit better.